src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcDatabaseMetaDataTest.java

```diff
@@ -122,6 +122,7 @@ private Column(
           new Column("ColTimestamp", Types.TIMESTAMP, "TIMESTAMP", 35, null, null, false, null),
           new Column("ColCommitTS", Types.TIMESTAMP, "TIMESTAMP", 35, null, null, false, null),
           new Column("ColNumeric", Types.NUMERIC, "NUMERIC", 15, null, 10, false, null),
+          new Column("ColJson", Types.NVARCHAR, "JSON", 2621440, null, null, false, 2621440),
```

Comment:
`2621440` is equal to the length of a `STRING(MAX)` column. If JSON has a different max length, then this should be changed.